pandas/io/stata.py

```diff
@@ -600,13 +606,12 @@ def _cast_to_stata_types(data: DataFrame) -> DataFrame:
         dtype = data[col].dtype
         for c_data in conversion_data:
             if dtype == c_data[0]:
-                # Value of type variable "_IntType" of "iinfo" cannot be "object"
-                if data[col].max() <= np.iinfo(c_data[1]).max:  # type: ignore[type-var]
+                if data[col].max() <= np.iinfo(c_data[1]).max:
                     dtype = c_data[1]
                 else:
                     dtype = c_data[2]
                 if c_data[2] == np.int64:  # Warn if necessary
-                    if data[col].max() >= 2**53:
+                    if data[col].max() >= 2 ** 53:
```

Comment:
You might need to update black to the most recent version. Newer versions of black do not add spaces here.

Reply:
Hm, it is indeed the case, these changes were made by black after running `black pandas/io/stata.py`. What surprises me though is that I build a docker image as specified in https://pandas.pydata.org/docs/development/contributing_environment.html and run the above command from within the container. Did I make something wrong in the setup, should I run some different command before pushing the code next time?

Reply:
It seems that `.pre-commit-config.yaml` is using a newer version of black than `requirements-dev.txt` and `environment.yml`. Feel free to open another PR to update `requirements-dev.txt` and `environment.yml` if you like.

Reply:
For now you can probably just run `pip install black==22.1.0` to fix your environment.

Reply:
Ok, will open another PR tomorrow to update `requirements-dev.txt` and `environment.yml`, and will soon update these lines in this PR. Thanks a lot for the review!